src/pages-helpers/curriculum/docx/curriculum-download-patcher/patches/cover.ts

```diff
@@ -0,0 +1,51 @@
+import type { Element } from "xml-js";
+
+import { CombinedCurriculumData } from "..";
+
+import { textIncludes, textReplacer } from "./util";
+
+function keyStageFromPhaseTitle(phaseTitle: string) {
```

Comment:
the only time where I think I would use a `case` and `switch` just for a bit more clarity, but that is being really nitpicky